src/models/ili9486.rs

```diff
@@ -146,30 +151,30 @@ where
 
 // common init for all color format models
 fn init_common<DELAY, DI>(
-    di: &mut DI,
+    dcs: &mut Dcs<DI>,
     delay: &mut DELAY,
     options: &ModelOptions,
-    colmod: u8,
-) -> Result<u8, Error>
+    colmod: Colmod,
+) -> Result<Madctl, Error>
 where
     DELAY: DelayUs<u32>,
     DI: WriteOnlyDataCommand,
 {
-    let madctl = options.madctl();
-    write_command(di, Instruction::SLPOUT, &[])?; // turn off sleep
-    write_command(di, Instruction::COLMOD, &[colmod])?; // 18bit 256k colors
-    write_command(di, Instruction::MADCTL, &[madctl])?; // left -> right, bottom -> top RGB
-    write_command(di, Instruction::VCMOFSET, &[0x00, 0x48, 0x00, 0x48])?; //VCOM  Control 1 [00 40 00 40]
-    write_command(di, Instruction::INVCO, &[0x0])?; //Inversion Control [00]
-    write_command(di, options.invert_command(), &[])?; // set color inversion
+    let madctl = Madctl::from(options);
+    dcs.write_command(Slpout)?; // turn off sleep
+    dcs.write_command(colmod)?; // 18bit 256k colors
+    dcs.write_command(madctl)?; // left -> right, bottom -> top RGB
+                                // dcs.write_command(Instruction::VCMOFSET, &[0x00, 0x48, 0x00, 0x48])?; //VCOM  Control 1 [00 40 00 40]
+                                // dcs.write_command(Instruction::INVCO, &[0x0])?; //Inversion Control [00]
```

Comment:
Should these be commented out?

Reply:
They were in the original C source I found at some point, but I don't think they are necessary.